### PR TITLE
Add script for identifying Jan-June 2018 master runs that are missing tags

### PR DESCRIPTION
**Pull request by stephenmcgruer**

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
@Hexcles @foolip FYI this is the 2018 version of the script for tagging master runs. The main changes from 2017 is that I use a graduated expectation of # tests, since the number of tests grows by about 4k during the time period and I didn't want to include e.g. bad results from June which have as few tests as results from January.

I only tackled Jan-June as there are essentially no tagless runs after that in 2018 (0-3 per month).

lmk if you have any concerns, otherwise I will go ahead with tagging these too. There's more of them this time! (~1k).

File: audit.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+/**
+ * Lists runs which do not have any of the three 'base' labels - master,
+ * pr_base, or pr_head. Such runs should be categorized into one of those three
```
Comment (foolip) [inline audit.js]: Some runs truly don't belong in any of the three buckets, namely full runs triggered from branches other than master.

Comment (stephenmcgruer) [reply audit.js]: Oh, I wonder if those are the 2019 runs I've been picking up. Perhaps runs from pushes to the `trigger/*` branches...

**Comment by stephenmcgruer** [comment]
This has been used to tag 1162 runs from Jan-June 2018.

[2018-runs-tagged-master.txt](https://github.com/foolip/ad-hoc-wpt-results-analysis/files/3928371/2018-runs-tagged-master.txt)

**Pull request closed without merging**